src/components/CreateGroupModal/CreateGroupModal.tsx

```diff
@@ -105,19 +106,19 @@ export const CreateGroupModal = ({ visible, onClose }: CreateGroupModalProps) =>
                         {...register('name', { required: tr('Required field') })}
                         error={errors.name}
                     />
-
-                    <FormRadio
-                        label={tr('Group type')}
-                        name="type"
-                        value={type}
-                        onChange={(v) => setType(v as GroupType)}
-                    >
-                        {groupTypes.map((t) => (
-                            <FormRadioInput key={t.type} value={t.type} label={t.text} />
-                        ))}
-                    </FormRadio>
-
-                    {nullable(type !== 'virtual', () => (
+                    {nullable(data?.access.group.create, () => (
+                        <FormRadio
+                            label={tr('Group type')}
+                            name="type"
+                            value={type}
+                            onChange={(v) => setType(v as GroupType)}
+                        >
+                            {groupTypes.map((t) => (
+                                <FormRadioInput key={t.type} value={t.type} label={t.text} />
+                            ))}
+                        </FormRadio>
+                    ))}
+                    {nullable(type !== 'virtual' && data?.access.group.create, () => (
```

Comment:
if you extract `data.access.group.create` into a variable, there will be a copypaste elimination + the code will look much better